Doc/whatsnew/3.9.rst

```diff
@@ -70,6 +70,17 @@ Summary -- Release highlights
 New Features
 ============
 
+Dictionary Merge & Update Operators
+-----------------------------------
+
+Merge (``|``) and update (``|=``) operators have been added to the built-in
+:class:`dict` class.
+
+.. XXX https://www.python.org/dev/peps/pep-0584/
+
+See :pep:`584` for a full description.
+
+(Contributed by Brandt Bucher in :issue:`36144`.)
```

Comment:
Hm, looking at the rendered docs, and comparing other sections, I wonder if this shouldn't just be a single paragraph.